src/exo/LoopIR_scheduling.py

```diff
@@ -3484,6 +3479,7 @@ class Schedules:
     DoExtractMethod = _DoExtractMethod
     DoParToSeq = _DoParToSeq
     DoReorderStmt = _DoReorderStmt
+    DoConfigWrite = _ConfigWrite
     DoConfigWriteAfter = _ConfigWriteAfter
```

Comment:
Shouldn't we remove ConfigWriteAfter?

Reply:
Probably; I'm trying to minimize changes to `LoopIR_scheduling.py` in this PR in order to make it easier to merge with other concurrent PRs.

Reply:
i.e. we should probably not remove dead code so much right now.